src/schema/agency-budget.ts

```diff
@@ -0,0 +1,16 @@
+import { pgTable, text } from "drizzle-orm/pg-core";
+import { z } from "zod";
+
+export const agencyBudget = pgTable("agency_budget", {
+  code: text("code"),
```

Comment:
code should be a [primary key](https://orm.drizzle.team/docs/indexes-constraints#primary-key)